libs/application/types/src/lib/template-api/shared-api/shared-api-definitions/payment-catalog.ts

```diff
@@ -6,5 +6,8 @@ export interface PaymentCatalogParameters {
 
 export const PaymentCatalogApi = defineTemplateApi<PaymentCatalogParameters>({
   action: 'paymentCatalog',
-  namespace: 'PaymentCatalog',
+  namespace: 'Payment',
+  params: {
+    organizationId: '6509142520',
```

Comment:
Is this default `organizationId` value or is the application template always required to provide the id when defining their paymentapi? Just wondering if we could accidentally have Syslumadur on some payments that should belong to another organisation?

Reply:
yeah good point. Ill remove the this id as the default and just throw an error if it is missing.